easycv/models/segmentation/encoder_decoder.py

```diff
@@ -55,12 +56,32 @@ def __init__(
         self.train_cfg = train_cfg
         self.test_cfg = test_cfg
 
-        self.init_weights(pretrained=pretrained)
-
-    def init_weights(self, pretrained=None):
-        if pretrained is not None:
-            print_log('load model from: {}'.format(pretrained), logger='root')
-        self.backbone.init_weights(pretrained=pretrained)
+        self.init_weights()
+
+    def init_weights(self):
+        logger = get_root_logger()
+        if isinstance(self.pretrained, str):
+            load_checkpoint(
+                self.backbone, self.pretrained, strict=False, logger=logger)
+        elif self.pretrained:
+            if self.backbone.__class__.__name__ == 'PytorchImageModelWrapper':
+                self.backbone.init_weights(pretrained=self.pretrained)
+            elif hasattr(self.backbone, 'default_pretrained_model_path'
```

Comment:
When is `default_pretrained_model_path` required?

Reply:
https://github.com/alibaba/EasyCV/blob/master/easycv/models/modelzoo.py